pybotframework/botstate.py

```diff
@@ -0,0 +1,34 @@
+
+
+class BotStateMachine(object):
```

Comment:
Nice way to keep compatibility with Python 2 (including "object").  Perhaps we should do this elsewhere you think?

Reply:
Yeah, we should do this everywhere we have a class.